dll/opengl/Mesa-GLU_for_ReactOS.txt

```diff
@@ -0,0 +1,85 @@
+Here is a brief explanation on how to synchronize Mesa/GLU in ReactOS.
```

Comment:
I'm not sure this would actually do much to help me do the next sync. It reads like a description of what you found when trying this, not a list of steps to follow.

Reply:
Indeed.
Shall I rework it a bit into "upstream re-discovery" part and "practical ROS update" part?
Or do you not want the "re-discovery" part in this doc at all and shall I move it to a Jira comment or even a Wiki page?

Reply:
Not sure if you saw this: https://github.com/reactos/reactos/commit/35e47afe0c6a7894a401cf8659e060a50c535fb1#commitcomment-25764998

Reply:
What's the status of the requested actions in this PR?